frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -129,8 +129,15 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
     onChange?.({ ...values, agencies: agencyIds });
   };
 
-  const resetFilter = () => {
-    ref.current.clear();
+  const resetFilter = (
+    setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void,
+  ) => {
+    ref.current?.getInstance()?.clear();
```

Comment:
Fixed react-bootstrap-typeahead error.
Added way to clear results that have not yet been searched for.